src/import/pull-tar.c

```diff
@@ -306,19 +359,14 @@ static void tar_pull_job_on_finished(PullJob *j) {
                                                             * in per-directory verification mode, since only
                                                             * then the signature is detached, and thus a file
                                                             * of its own. */
-                        log_error_errno(j->error, "Failed to retrieve signature file, cannot verify. (Try --verify=no?)");
-                        r = i->signature_job->error;
+                        r = log_error_errno(i->signature_job->error,
+                                            "Failed to retrieve signature file, cannot verify. (Try --verify=no?)");
```

Comment:
Is actively suggesting to disable validation a good idea? It needs to be available ofc, but perhaps it should not be encouraged

Reply:
Well, I put it in question form and in brackets, to distance myself from this idea. But reality is that it's extremely useful, so I fiugred i might be something to help people with.

And there are OK usecases (for example, if you know the hash beforehand and pass it in, or if you validate externally or so. I think being too "judgy" is not helpful.

I mean I added this originally myself since I kept forgetting what the right switch was after not looking at the code for a bit, so this is primarily there to help myself ;-)

Reply:
makes sense